src/githubWrapper.js

```diff
@@ -43,7 +89,7 @@ function getRemoteRepo() {
 }
 
 function getDefaultBranch(repo) {
-    return github.repos.get(repo)
+    return wrapper.ghApi().repos.get(repo)
         .then((response) => {
             return response.data.default_branch;
         });
```

Comment:
If I'm reading this right, shouldn't every use of `ghApi` use `checkFor2faError` on error?  If so, it looks like this one was missed.

Reply:
good catch. i'll fix it